### ISSUE TITLE
Problems with headfake.transformer.IntermittentBlanks

### ISSUE THREAD

**Issue by DVDLL** [rcfgroup/headfake#18]
Hello, I'm having problems with the class _headfake.transformer.IntermittentBlanks_ while I try the tutorial 2.
When I add these lines indicated in the tutorial,

```
- name: last_name
  class: headfake.field.LastNameField
  gender_field: gender
  transformers:
  - class: headfake.transformer.IntermittentBlanks
    blank_probability: 0.2
    blank_value: NULL
  - class: headfake.transformer.UpperCase
```

I obtain the next error:

`headfake.error.TransformerError: Error transforming 'last_name' value. Transformer: '<class 'headfake.transformer.IntermittentBlanks'>'; data:{'main_pat_id': 'S1000000', 'gender': 'M', 'first_name': 'Leonard'}; Original error:TypeError (IntermittentBlanks.transform() takes 3 positional arguments but 4 were given)`

The rest of the code is as the tutorial indicates.
I don't know why this occurs, if someone could help me I would be very grateful.

**Comment by rcfree** [comment]
Sorry to hear you have been having problems. 

This should be fixed in the latest version (v1.1.1) which you should be able to upgrade to using pip with: `pip install -U headfake`.

Please let me know if you are still having problems.

**Comment by DVDLL** [comment]
Hello, I upgraded and now it works, thank you!
